apps/application-system/form/infra/application-system-form.ts

```diff
@@ -27,6 +27,16 @@ export const serviceSetup = (services: {
       SI_PUBLIC_DD_RUM_APPLICATION_ID: '/k8s/DD_RUM_APPLICATION_ID',
       SI_PUBLIC_DD_RUM_CLIENT_TOKEN: '/k8s/DD_RUM_CLIENT_TOKEN',
     })
+    .resources({
+      limits: {
+        cpu: '200m',
+        memory: '256Mi',
```

Comment:
Is this enough? Seems pretty low for a limit

Reply:
What are you basing this on?